Source/rFactor2SharedMemoryMap.cpp

```diff
@@ -39,12 +39,12 @@ Shared resources:
     * RulesControl - mapped view of rF2RulesControl structure
     * PluginControl - mapped view of rF2PluginControl structure
 
-  Aside from Extended (see below), output buffers are (with few exceptions) exact mirror of ISI structures, plugin constantly memcpy'es them
+  Aside from Extended (see below), output buffers are (with few exceptions) exact mirror of ISI structures, plugin constantly memcpy's them
   from game to memory mapped files.
 
   Plugin offers optional weak synchronization by using version variables on each of the output buffers.
 
-  Input buffers are meant to be filled out by the clients.  To avoid complex locking input buffers use version variables as well, and were
+  Input buffers are meant to be filled out by the clients.  To avoid complex locking input buffers use they version variables as well, and were

```

Comment:
their?

Reply:
Perhaps, probably, maybe.  I could offer arguments both ways, go with their.

Reply:
Just triple checking - is "they" in the right place in a sentence?

Reply:
Oops.  ..._they use version_...